src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -417,8 +442,10 @@ public void shouldSendSubscriptionUpdatedNotificationForSubscriptionUpdatedReque
     @Test
     public void shouldSendSubscriptionUpdatedNotificationForSubscriptionUpdatedRequestWithNewEmailAddressForAPaperHearing() throws Exception {
         updateJsonForPaperHearing();
-        json = json.replace("appealReceived", "subscriptionUpdated");
-        json = updateEmbeddedJson(json, "No", "case_details", "case_data", "subscriptions", "appellantSubscription", "subscribeSms");
+        json = updateEmbeddedJson(json, "subscriptionUpdated", "event_id");
+        json = updateEmbeddedJson(json, "paper", "case_details", "case_data", "appeal", "hearingType");
```

Comment:
Looks like maybe you are doing the same thing twice now. `updateJsonForPaperHearing()` also updates this line in json